### ISSUE TITLE
Spoon report and screenshots

### ISSUE THREAD

**Issue by francescogatto** [sebaslogen/CleanGUITestArchitecture#24]
Hi! nice test application, but what's about the Spoon report? Is it generated? Thanks

**Comment by sebaslogen** [comment]
No, the Spoon library here is only used to take screenshots, not to run or generate test reports.
Cheers.

**Comment by francescogatto** [comment]
Do You know if is it possible to have spoon reports with cucumber?

**Comment by sebaslogen** [comment]
I don't think so out of the box, because both cucumber and Spoon take control of how the tests batch is divided and executed
